src/ClientWidgetApi.ts

```diff
@@ -601,17 +601,30 @@ export class ClientWidgetApi extends EventEmitter {
         const isDelayedEvent = request.data.delay !== undefined || request.data.parent_delay_id !== undefined;
         if (isDelayedEvent && !this.hasCapability(MatrixCapabilities.MSC4157SendDelayedEvent)) {
             return this.transport.reply<IWidgetApiErrorResponseData>(request, {
-                error: { message: "Missing capability" },
+                error: { message: `Missing capability for ${MatrixCapabilities.MSC4157SendDelayedEvent}` },
+            });
+        }
+
+        const isStickyEvent = request.data.sticky_duration_ms !== undefined;
```

Comment:
I am surprised, isn't sticky event a new `IXXXFromWidgetActionRequest`? it is re-using an existing AI and adding the sticky duration in the data?
Can this create some sort of collision of event content and the actual request?
I think that sending sticky event is a query param, not adding sitcky duration in the payload?